src/registrar/tests/common.py

```diff
@@ -1,7 +1,57 @@
+import os
+import logging
+
+from contextlib import contextmanager
```

Comment:
**question**: It seems very unfortunate to have to repeat this exact code across the `api` and `registrar` Django apps. Is it possible to define it in one of the apps and find an appropriate `import` statement to bring it into the other one? If not, can we put it somewhere else and use it from there in both apps?

Reply:
I considered that because I am also not a fan of the code duplication, but Django apps are intended to be plug-and-play with each other. So I think avoiding non-functional dependencies between them is the lesser of two evils in this case.